src/postgraphql/http/__tests__/setupServerSentEvents-test.js

```diff
@@ -0,0 +1,54 @@
+import setupServerSentEvents from '../setupServerSentEvents'
+
+const http = require('http')
+const EventEmitter = require('events') // tslint:disable-line:variable-name
+const request = require('supertest-as-promised')
+
+const _emitter = new EventEmitter()
+let connection
+
+const requestHandler = async (req, res, next) => {
+  const options = { _emitter, watchPg: true }
+  connection = req
+  setupServerSentEvents(req, res, options)
+}
+
+const createServer = () => http.createServer(requestHandler)
+
+test('will set the appropriate headers', async () => {
+  const server = createServer()
+  setTimeout(() => connection.emit('close'), 25)
```

Comment:
https://travis-ci.org/calebmer/postgraphql/jobs/194803070

Seems this 25 millisecond timeout is not long enough for the tests:

```
TypeError: Cannot read property 'emit' of undefined
    at Timeout._onTimeout (/home/travis/build/calebmer/postgraphql/src/postgraphql/http/__tests__/setupServerSentEvents-test.js:26:59)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
 FAIL  src/postgraphql/http/__tests__/setupServerSentEvents-test.js
  ● Test suite failed to run
    ProcessTerminatedError: cancel after 2 retries!
      
      at Farm.<anonymous> (node_modules/worker-farm/lib/farm.js:81:25)
      at Array.forEach (native)
      at Farm.<anonymous> (node_modules/worker-farm/lib/farm.js:75:36)
      at ontimeout (timers.js:365:14)
      at tryOnTimeout (timers.js:237:5)
      at Timer.listOnTimeout (timers.js:207:5)
```

I think what's happening is the `requestHandler` is not called by the time the timeout fires, meaning `connection` is not defined and hence `connection.emit('close')` throws.